Sources/App/Controllers/MeetupController.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  ProfileController.swift
+//  App
+//
+//  Created by  LaptopGCampos on 7/9/18.
+//
+
+import Vapor
+import Fluent
+
+struct MeetupController: RouteCollection {
+    
+    func boot(router: Router) throws {
+        let publicRoutes = router.grouped("api", "meetup")
+        publicRoutes.get("currentSpeakers", use: currentSpeakers)
+    }
+    
+    func currentSpeakers(_ request: Request) throws -> Future<[Profile]> {
+        return Meetup.query(on: request).filter(\.statusID == Status.scheduled.rawValue).first()
```

Comment:
most of the time it is better to have fat models and thin controllers, this filtering is better in a `Meetup` method than here, so it would be like:

`Meetup.query(on: request).firstScheduled()`

A controller involves business logic, and no data logic :ponteverga:

Reply:
cool! I like this approach. thanks.